mtoken-model/src/main/java/com/wultra/security/powerauth/lib/mtoken/model/entity/attributes/Attribute.java

```diff
@@ -86,7 +87,12 @@ public enum Type {
         /**
          * Image attribute type - represents an image as a URL to the thumbnail and the higher resolution.
          */
-        IMAGE
+        IMAGE,
+
+        /**
+         * Customization texts to show for {@code success}, {@code failure}, or {@code reject} operation.
+         */
+        RESULT_TEXTS
```

Comment:
ℹ️ It should not be needed to model the discriminator, but let's refactor that in a separate issue.